tensorflow/tools/docker/Dockerfile.gpu

```diff
@@ -1,9 +1,10 @@
-FROM nvidia/cuda:7.5-cudnn4-runtime
+FROM nvidia/cuda:7.5-cudnn4-devel
```

Comment:
Really? Which tests depends on it? I thought it runs python scripts. The devel stuff should be only needed if you compile some c/c++ code. And that should not be happening.

If it really is a problem for a few tests then I would ignore them here.